src/rules/noWeakTypes.js

```diff
@@ -0,0 +1,31 @@
+import {get} from 'lodash';
```

Comment:
To keep inline with the rest of the codebase, please change this to:

```js
import _ from 'lodash';
```

We are using https://github.com/lodash/babel-plugin-lodash to rewire them in the build step.